docs/int/key-concepts.md

```diff
@@ -85,6 +85,16 @@ A distributed validator cluster is a collection of distributed validator nodes c
 
 A distributed validator key is a group of BLS private keys, that together operate as a threshold key for participating in proof of stake consensus with.
 
+| Cluster Size | Threshold | Note              |
```

Comment:
```suggestion
The number of nodes in a cluster that need to be online and honest for their distributed validators to be online is outlined in the following table.

| Cluster Size | Threshold | Note              |
```

Can't tell whether this should be an *is* outlined in the following or *are* outlined in the following...

Reply:
I think "is" makes sense in this case.